postgresql/src/test/scala/akka/persistence/postgresql/journal/PostgreSqlJournalSpec.scala

```diff
@@ -27,34 +27,42 @@ import scala.concurrent.duration._
 
 object PostgreSqlJournalSpec {
 
-  private val JournalPluginConfig = ConfigFactory.parseString(
-    """
-      |akka.persistence.journal.plugin = "postgresql-journal"
-      |akka.loglevel = "DEBUG"
-      |""".stripMargin)
+  private val JournalPluginConfig = ConfigFactory.parseString("""
+                                                                |akka.persistence.journal.plugin = "postgresql-journal"
+                                                                |akka.loglevel = "DEBUG"
+                                                                |""".stripMargin)
 }
 
 /**
  * Test case for [[PostgreSqlJournal]].
  */
 final class PostgreSqlJournalSpec
     extends JournalSpec(config = PostgreSqlJournalSpec.JournalPluginConfig)
-        with Eventually with ProjectionExtensionSpec {
+    with Eventually
+    with ProjectionExtensionSpec {
 
   protected val r2dbc: R2dbc = R2dbc(
-    new PostgresqlConnectionFactory(PostgresqlConnectionConfiguration.builder()
+    new PostgresqlConnectionFactory(
+      PostgresqlConnectionConfiguration
+        .builder()
         .host(system.settings.config.getString("postgresql-journal.db.hostname"))
         .username("postgres")
         .password("s3cr3t")
         .database("db")
-        .build())
+        .build()
+    )
   )
 
   override def beforeAll(): Unit = {
     eventually(timeout(60.seconds), interval(1.second)) {
-      r2dbc.withHandle(handle => handle.executeQuery(Schema.SQL + "TRUNCATE event, tag; " +
-          "CREATE TABLE IF NOT EXISTS projected (id varchar(255), value text);", _.getRowsUpdated
-      )).blockLast()
+      r2dbc
+        .withHandle(handle =>
+          handle.executeQuery(Schema.SQL + "TRUNCATE event, tag; " +
```

Comment:
The indentation looks odd like this way Move the SQL to a variable above the eventually call.  Do the same for the MySql specs too.